RInAppMessaging/Classes/Enums/Public/Identification.swift

```diff
@@ -1,6 +1,5 @@
 /// Internal enum for encoding user identifiers
-@objc public enum Identification: Int, Codable {
-    case rakutenId = 1
```

Comment:
Maybe we need to check that no customer is relying on this id before removing it.

Reply:
Even if they do, they should switch to userId. On backend side userId and rakutenId are treated the same. In the dashboard you provide the list of rakuten ids as user ids. Maybe it's good to mention that in README in migration guide or something like that.

Reply:
OK. Yes it'll need a migration guide. The UserInfoProvider change would require a major version anyway so maybe better to include this removal now.